src/docs/product/alerts/alert-types.mdx

```diff
@@ -58,26 +58,61 @@ When you create an alert, all the displayed alert types (except “Issues”) ma
 
 ### Errors
 
-- Number of Errors
-- Users Experiencing Errors
+Error alerts are useful for monitoring the overall level of errors in your project, errors occurring in specific parts of your app, or errors affecting your users.
+
+- **Number of Errors**
+  Alerts when the number of errors in a project matching your filters crosses the threshold you set.
+
+- **Users Experiencing Errors**
+  Alerts when the number of users affected by errors in your project crosses the threshold you set.
 
 ### Sessions (Crash Rate Alerts)
 
-Crash rate alerts tell you when the crash free percentage for either [sessions or users](/product/releases/health/#active-sessionsusers) falls below a specific threshold. This could happen because of a spike in the number of session or user [crashes](/product/releases/health/#crash). Use crash rate alerts to monitor the health of your application per project, or in a specific release
+Crash rate alerts can give you a better picture of the health of your app on a per project basis or in a specific release. They trigger when the crash-free percentage for either [sessions or users](/product/releases/health/#active-sessionsusers) falls below a specific threshold. This could happen because of a spike in the number of session or user [crashes](/product/releases/health/#crash).
+
+- **Crash Free Session Rate**
```

Comment:
Since these aren't headers of any sort, I assume we can't link to them right? In that case, are we thinking of just linking users to this page from the API documentation (as opposed to linking each metric alert type directly to its section on this page)?

Reply:
This is a good point - we could add anchors to these bullets so we can link to them directly.

Reply:
Hi there, great question. I think we should link the H3 Metric Alert type headers, so Errors, Sessions, and Performance. What do you think?

Reply:
I think that'd be the next best solution if we can't add anchors to the bullets!